tests/e2e/e2e_http_ingress_test.go

```diff
@@ -0,0 +1,135 @@
+package e2e
+
+import (
+	"context"
+	"net/http"
+	"time"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"helm.sh/helm/v3/pkg/action"
+	"helm.sh/helm/v3/pkg/chart/loader"
+	helmcli "helm.sh/helm/v3/pkg/cli"
+	"k8s.io/api/extensions/v1beta1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/intstr"
+
+	. "github.com/openservicemesh/osm/tests/framework"
+)
+
+var _ = OSMDescribe("HTTP ingress",
+	OSMDescribeInfo{
+		Tier:   1,
+		Bucket: 1,
+	},
+	func() {
+		const destNs = "server"
+
+		It("allows HTTP ingress traffic", func() {
+			// Install OSM
+			installOpts := Td.GetOSMInstallOpts()
+			Expect(Td.InstallOSM(installOpts)).To(Succeed())
+
+			Expect(Td.CreateNs(destNs, nil)).To(Succeed())
+			Expect(Td.AddNsToMesh(true, destNs)).To(Succeed())
+
+			// Get simple pod definitions for the HTTP server
+			svcAccDef, podDef, svcDef := Td.SimplePodApp(
+				SimplePodAppDef{
+					Name:      "server",
+					Namespace: destNs,
+					Image:     "kennethreitz/httpbin",
+					Ports:     []int{80},
+				})
+
+			_, err := Td.CreateServiceAccount(destNs, &svcAccDef)
+			Expect(err).NotTo(HaveOccurred())
+			_, err = Td.CreatePod(destNs, podDef)
+			Expect(err).NotTo(HaveOccurred())
+			_, err = Td.CreateService(destNs, svcDef)
+			Expect(err).NotTo(HaveOccurred())
+
+			// Expect it to be up and running in it's receiver namespace
+			Expect(Td.WaitForPodsRunningReady(destNs, 60*time.Second, 1)).To(Succeed())
+
+			// Install nginx ingress controller
+			helm := &action.Configuration{}
+			Expect(helm.Init(Td.Env.RESTClientGetter(), Td.OsmNamespace, "secret", Td.T.Logf)).To(Succeed())
+			install := action.NewInstall(helm)
+			install.RepoURL = "https://kubernetes.github.io/ingress-nginx"
```

Comment:
This is interesting. Will make us keep our solutions up to date with a live nginx ingress chart. Cool.